cdk/apps/rnasum/app.py

```diff
@@ -2,7 +2,36 @@
 from aws_cdk import core
 from stacks.iap_tes import IapTesStack
 from stacks.cicd import CICDStack
+from stacks.batch import BatchStack
+from stacks.common import CommonStack
 
+rnasum_image = 'umccr/rnasum:0.3.5'
+
+dev_env = {'account': '843407916570', 'region': 'ap-southeast-2'}
```

Comment:
Might be worth putting this higher on a `context.json` file as Alexis found out?

Reply:
There is a general lack of definition how we handle deployment to different accounts... 

Again, one for when we have something deployed?

Reply:
Follow the lead here. But, yes, feel something missing like TF `workspace`. Need deep dive CDK a bit more on this, for me, on how to isolate prod, dev environments better. Could do improvement in next iteration.